package.json

```diff
@@ -29,7 +29,7 @@
         "build": "npm run clean && npm run generate:icons && npm run build:library && npm run build:extractedThirdPartyStyles",
         "build:extractedThirdPartyStyles": "concat src/components/**/*.css | postcss > dist/extractedThirdPartyStyles.css",
         "build:library": "vite build",
-        "build:storybook": "NODE_OPTIONS=--max_old_space_size=6144 build-storybook",
+        "build:storybook": "storybook build",
```

Comment:
We added `NODE_OPTIONS=--max_old_space_size=6144` because we were running out of memory in huge PR like re rendering all icons. Should we delete it? 🤔

Reply:
Let's add it back if it bugs us?

Reply:
Yes, we can do that!